app/components/badge_component.scss

```diff
@@ -5,3 +5,17 @@
 .lg-verification-badge .usa-icon {
   margin-right: units(1);
 }
+
+.border-warning {
```

Comment:
`.border-warning` already exists, as [implemented by the design system](https://designsystem.digital.gov/utilities/border/#utility-border-color), so I don't think we need to duplicate it here, and I wouldn't expect the utility class to customize icon or text color.

Instead, I think we could apply classes `border-warning text-warning` on the HTML elements and avoid this additional CSS altogether.